test/test_automated_install.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from textwrap import dedent
+
+SETUPVARS = {
```

Comment:
This is the 'expected' side of the test.  we'll need to have it match the case of the setupVars.txt too.